KsApi/Service+RequestHelpers.swift

```diff
@@ -64,4 +51,35 @@ extension Service {
       .rac_dataResponse(preparedRequest(forURL: paginationUrl), and: self.perimeterXClient)
       .flatMap(self.decodeModelToSignal)
   }
+
+  func requestPaginationDecodable<M: Decodable>(_ paginationUrl: String)
```

Comment:
What does `M` represent here?

Reply:
`M` is the generic type placeholder for the model that will be decoded, for example, a `Project` or a `Backing`.

Reply:
Got it. I figured! I would prefer Model here but totally up to you. Just a nit 🙂